backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -747,12 +836,8 @@ export class UserResolver {
   @Query(() => Boolean)
   async hasElopage(@Ctx() context: Context): Promise<boolean> {
     logger.info(`hasElopage()...`)
-    const userEntity = context.user
-    if (!userEntity) {
-      logger.info('missing context.user for EloPage-check')
-      return false
-    }
-    const elopageBuys = hasElopageBuys(userEntity.email)
+    const userEntity = getUser(context)
+    const elopageBuys = hasElopageBuys(userEntity.emailContact.email)
```

Comment:
Why do we not need this check anymore?

Reply:
I don't know, if it is still in use from the FE, so i refactored it...